src/main/java/examples/Examples.java

```diff
@@ -803,4 +803,25 @@ public void collector02Example(PgClient client) {
         }
       });
   }
+
+  public void cancelRequest(PgConnection connection) {
+    connection.query("SELECT pg_sleep(20)", ar -> {
+      if (ar.succeeded()) {
+        // imagine this is a long query and is still running
+        System.out.println("Query success");
```

Comment:
no println please

Reply:
I think however that the test should exit with a response from this part, using an Async from vertx-unit

Reply:
ah sorry it is example :-)